ui/src/virtualconsole/vccuelist.cpp

```diff
@@ -877,6 +881,7 @@ void VCCueList::slotFunctionRunning(quint32 fid)
     else if (playbackLayout() == PlayStopPause)
         m_playbackButton->setIcon(QIcon(":/player_stop.png"));
     m_timer->start(PROGRESS_INTERVAL);
+    emit buttonAppearanceChanged();
```

Comment:
I meant this is not needed and related slot

Reply:
@mcallegari If we remove this signal, the playback button icon will not sync.

Reply:
@mcallegari 
Of course, there are other ways to sync the button icons but the current version is very simple. 
Your thought?

Reply:
Maybe we're just not using the proper terminology.
One thing is the layout change (Play + Stop vs Play + Pause), which is not needed during operate mode.
One other thing is the playback status change, which highlights the buttons with different colors.
Since you named this "buttonAppearanceChanged" I assumed it was the first case.
Then please rename it to "playbackStatusChanged"